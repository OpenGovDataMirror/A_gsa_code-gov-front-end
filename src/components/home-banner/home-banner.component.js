/* global PUBLIC_PATH */

import React, { Fragment } from 'react'
import CustomLink from 'components/custom-link'
import HomeBannerSearchBox from '../home-banner-search-box'

export default class HomeBanner extends React.Component {
  componentDidMount() {}

  render() {
    return (
      <section id="banner-home">
        <div className="bg-primary text-center">
          <div className="padding-y-205 tablet:padding-y-4">
            <div className="font-heading-xl tablet:font-heading-2xl text-white line-height-sans-2">
              {this.props.motto}
            </div>
            <div className="font-heading-lg text-white desktop:display-block display-none margin-top-1">
              {this.props.subtitle}
            </div>
          </div>
        </div>
      </section>
    )
  }
}
