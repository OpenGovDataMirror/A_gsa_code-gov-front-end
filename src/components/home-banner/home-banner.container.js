import { connect } from 'react-redux'
import { push } from 'connected-react-router'
import { getConfigValue, normalize, now } from 'utils/other'
import HomeBannerComponent from './home-banner.component'

export const mapStateToProps = () => ({
  motto: getConfigValue('content.home.banner.motto'),
  subtitle: getConfigValue('content.home.banner.subtitle')
})

export default connect(mapStateToProps)(HomeBannerComponent)
