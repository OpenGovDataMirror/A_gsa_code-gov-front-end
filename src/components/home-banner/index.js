import HomeBannerContainer from './home-banner.container'

export default HomeBannerContainer
