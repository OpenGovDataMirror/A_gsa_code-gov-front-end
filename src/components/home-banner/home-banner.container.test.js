import { mapStateToProps, mapDispatchToProps } from 'components/home-banner/home-banner.container'

describe('containers - HomeBanner', () => {
  describe('mapStateToProps', () => {
    it('should return the correct properties', () => {
      expect(mapStateToProps()).toMatchSnapshot()
    })
  })
})
