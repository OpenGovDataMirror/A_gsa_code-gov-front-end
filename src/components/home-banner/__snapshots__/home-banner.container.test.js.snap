// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers - HomeBanner mapStateToProps should return the correct properties 1`] = `
Object {
  "motto": "Sharing America's Code",
  "subtitle": "Unlock the tremendous potential of the Federal Governmentâ€™s software.",
}
`
